backend/src/test/java/com/example/backend/controller/PetControllerTest.java

```diff
@@ -65,12 +65,31 @@ void when_addPet_then_OK() throws Exception {
     void when_addPet_with_NotValidName_then_BadRequest() throws Exception {
         mockMvc.perform(MockMvcRequestBuilders.post("/api/pets/")
                         .contentType(MediaType.APPLICATION_JSON).content("""               
-                                {"name":"","nameOfBreed":"albino", "photo":"albino.png","supplies": ["Water Bottle","Roomy Cage"] }
+                                {"name":"","nameOfBreed":"albino","photo":"albino.png","supplies": ["Water Bottle","Roomy Cage"]}
                                     """))
                 .andExpect(status().isBadRequest());
 
     }
 
+    @Test
+    @DirtiesContext
+    void when_getPetById_then_OK() throws Exception {
+        petRepo.save(pet1);
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/pets/1"))
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
+                        {"id":"1","name":"Whiskers","nameOfBreed":"albino", "photo":"albino.png","supplies": ["Water Bottle","Roomy Cage"]}          
+                        """));
```

Comment:
This is difficult to read. I suggest to put each json-property in one line.

Reply:
Thanks a lot :-)